src/main/Extensions/ApplicationSearch/ApplicationSearch.ts

```diff
@@ -67,8 +67,8 @@ export class ApplicationSearch implements Extension {
             "en-US": {
                 extensionName: "Application Search",
                 searchResultItemDescription: "Application",
-                openApplication: "Open application",
-                openApplicationAsAdministrator: "Open application as administrator",
+                openApplication: "Open",
+                openApplicationAsAdministrator: "Open as admin",
```

Comment:
@tkohlmeier FYI I shortened these because the action menu is getting a bit long with the additional keyboard shortcuts. Maybe we have to check more actions and shorten the translations.